serialization/python_pickle.ksy

```diff
@@ -0,0 +1,448 @@
+meta:
+  id: python_pickle
+  title: Python pickle serialization format (protocol 3)
+  application: Python
+  file-extension:
+    - pickle
+    - pkl
```

Comment:
The Pickle format has a [Wikidata entry](https://www.wikidata.org/wiki/Q7190889) and a [JSTFFP page](http://fileformats.archiveteam.org/wiki/Pickle), these could be added in an [`xref` block](http://doc.kaitai.io/user_guide.html#meta-xref).